decidim-forms/app/models/decidim/forms/question.rb

```diff
@@ -14,8 +14,28 @@ class Question < Forms::ApplicationRecord
                dependent: :destroy,
                inverse_of: :question
 
+      has_many :display_conditions,
+               class_name: "DisplayCondition",
+               foreign_key: "decidim_question_id",
+               dependent: :destroy,
+               inverse_of: :question
+
+      has_many :display_conditions_for_other_questions,
```

Comment:
So this associates display conditions to this question, but the name says "for other questions". Can you document what or which are "other questions" please?